MovieKuno/Tab Pages/Home Views/MovieCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MovieCell.swift
+//  MovieKuno
+//
+//  Created by Leah Joy Ylaya on 1/21/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct MovieCell: View {
+    var movie: Movie!
+    var body: some View {
+        VStack {
+            Image(Asset.moviePlaceholder.name)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .frame(width: 130, height: 200)
+                .cornerRadius(20)
+            Text(movie.title)
+                .font(.system(size: 17))
+                .bold()
+                .foregroundColor(Color(Asset.textColorMain.color))
```

Comment:
By the way if this kind of style will be used in the future, maybe consider creating a style for it? ref: https://www.swiftbysundell.com/articles/encapsulating-swiftui-view-styles/